app/assets/javascripts/Grader/marking.js

```diff
@@ -197,15 +197,28 @@ function expand_unmarked(elem, criterion_class) {
     } else {
       hide_criterion(parseInt(elem.getAttribute('data-id'), 10), criterion_class);
     }
-  } else {
+  } else if (criterion_class == 'FlexibleCriterion') {
     if (elem.value == '') {
       show_criterion(parseInt(elem.getAttribute('data-id'), 10), criterion_class);
     } else {
       hide_criterion(parseInt(elem.getAttribute('data-id'), 10), criterion_class);
     }
+  } else {
+    // We have two radio buttons nested in this block that we must evaluate.
+    var anyRadioButtonSet = false;
+    jQuery(elem).find('.mark_grade_input_checkbox').each(function(index, element) {
+      if (element.checked) {
+          anyRadioButtonSet = true;
+      }
```

Comment:
Eh, you should be able to put the `checked` into the selector itself and then see if the `find` returns anything or not.